### PR TITLE
asadiqbal08/Addressed Resource Page Feedback

### PR DESCRIPTION
**Pull request by asadiqbal08**
#### What are the relevant tickets?
fixes: #693 
[Trello Ticket](https://trello.com/c/YGrEvjcw/186-resource-page-feedback)

#### What's this PR do?
Updated the resource pages as requested.

#### Screenshots (if appropriate)
screencapture-bc-odl-local-8099-term-service-2020-06-23-12_56_19

screencapture-bc-odl-local-8099-term-service-2020-06-23-12_56_41

screencapture-bc-odl-local-8099-term-service-2020-06-23-12_57_10

### CODE REVIEW

File: cms/models.py [hunk 1]
```diff
@@ -459,11 +459,13 @@ class ResourcePage(Page):
     template = "resource_template.html"
     parent_page_types = ["HomePage"]
 
-    sub_heading = models.CharField(
-        max_length=250,
+    header_image = models.ForeignKey(
+        Image,
         null=True,
```
Comment (umarmughal824) [inline cms/models.py:464]: This can't be blank or optional, as it does not show same in the template.

File: cms/templates/resource_template.html [hunk 2]
```diff
@@ -9,32 +9,38 @@
 {% endblock %}
 
 {% block content %}
-<section class="container resource-container">
-  <h1> {{ page.title }} </h1>
+<section class="resource-container">
+  <div class="banner-holder">
+    <img class="resource-image" src="{% image_version_url page.header_image 'fill-1920x540' %}" alt="Resource Page" />
```
Comment (umarmughal824) [inline cms/templates/resource_template.html:14]: If it's optional in the model, then why not here?

File: static/scss/resource.scss [hunk 3]
```diff
@@ -5,11 +5,172 @@ html {
...
+      background: $white;
+      content: "";
+
+      @include media-breakpoint-down(sm) {
+        left: 5px;
+        right: 5px;
+      }
+    }
+
+    &.singal-child {
```
Comment (pdpinch) [inline static/scss/resource.scss:59]: perhaps this should be `single-child` ? 

```suggestion
    &.single-child {
```

**Comment by pdpinch** [comment]
In the screenshots, it looks like the margin above "back to top" is the same as the margin below. This makes it hard to tell what text the "back to top" link is associated with. 

Can we reduce the margin above "back to top" ?

**Comment by asadiqbal08** [comment]
updated. @pdpinch @umarmughal824 

screencapture-bc-odl-local-8099-term-service-2020-06-23-18_06_20

**Comment by umarmughal824** [comment]
@pdpinch could you please review the updated screenshot?

**Pull request merged**